php/router.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// Used by `wp server` to route requests.
+
+namespace WP_CLI\Router;
+
+function get_full_host( $url ) {
```

Comment:
Isn't this the full host, plus optional port? Why does it need to be a utility method?